src/common/components/storage/DisksTable.tsx

```diff
@@ -29,11 +29,11 @@ interface DisksTableProps extends WithTestID {
   updateDiskSkipFormatting?: DiskFormattingType;
 }
 
-const diskColumns = [
+const diskColumns = (showFormat: boolean) => [
```

Comment:
```suggestion
const diskColumns = (showFormat: boolean) => [
  { title: 'Name' },
  { title: 'Role' },
  { title: 'Limitations' },
  showFormat ? { title: 'Format?' } : false,
  { title: 'Drive type' },
  { title: 'Size' },
  { title: 'Serial' },
  { title: 'Model' },
  { title: 'WWN' },
].filter(Boolean);
```

Reply:
Maybe not... 
![image](https://user-images.githubusercontent.com/87187179/224647147-5f57917e-abfc-4840-9b5d-7e4848fbbf9e.png)

`.filter(Boolean)` changes the type of the array and PT table doesn't like that. The current version works and I would keep it as is.